src/test/java/TestEditRulePopup.java

```diff
@@ -26,44 +29,65 @@ void setUp() throws Exception {
 		addpopup = new EditRulePopup(new CodeQualityRule("", "LAA < 5", false, false));
 	}
 
+	/**
+	 * Tests getConditon() method
+	 */
 	@Test
 	void testGetCondition() {
 		addpopup.getCondition();
 	}
-
+	
+	/**
+	 * Tests getValue() method

```

Comment:
Tests if the EditRulePopup returns the correct whatever - analisa o getValue() and please update accordingly